homeassistant/components/xiaomi_miio/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Xiaomi Miio",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/xiaomi_miio",
-  "requirements": ["construct==2.9.45", "python-miio==0.5.0.1"],
+  "requirements": ["zeroconf==0.27.1", "construct==2.9.45", "python-miio==0.5.0.1"],
```

Comment:
I wonder if we should do this or do an `after_dependencies` ? That will also ensure that we install the `zeroconf` requirements.

Reply:
Should this be in `setup.py` instead ?

Reply:
we only put things in `setup.py` that are used by core/helpers/util. Things that are integration specific are not added. We do have the contraints file to make sure that requirements of any integration that are part of the default config will be constrained to their required version.

Reply:
`after_dependencies` makes more sense in that case as we don't want to have to update the version everywhere.  I'll modify the pr

Reply:
done